README.md

```diff
@@ -1,119 +1,109 @@
+	1. Navigate to root directory 'C:\Users\Username'. `cd ~`
+  2. In root directory (~), `mkdir .ssh`
+  3. `ls .ssh`
+  4. Inside `~/.ssh`, `touch config`
+2. Your interviewer will send you some connection information to paste into this newly created config file. Open it up, paste the connection info, then save. Example:
+    ```
+    Host hc-interview
     HostName EC2_INSTANCE_URL_GOES_HERE
     User candidate
     LocalForward 3000 localhost:3000
-    LocalForward 3001 localhost:3001```
-- Open VS Code
-- Press `F1` and enter `Remote-SSH: Connect to Host`
-- Select `hc-interview`
-- A new VS Code tab will open and ask you to enter a password in the terminal. The password is `candidate`
-- Once entered, select `Open Folder` from the left side of the screen
-- Select `/home/candidate/interview-internal-apps-2022/` from the dropdown
-- You will be asked to enter the password again. `candidate`
-- You should now be able to see the files and code base we will be working in on the left side of the screen. Any code changes you make and save, your interviewers will be able to see. 
+    LocalForward 3001 localhost:3001
+    ```
+3. Open VS Code, then press `F1` and enter `Remote-SSH: Connect to Host`
+4. Select `hc-interview`. A new VS Code tab will open and ask you to enter a password in the terminal. The password is `candidate`.
+5. Select `Open Folder` from the left side of the screen.
+6. Select `/home/candidate/interview-internal-apps-2022/` from the dropdown. You will be asked to enter the password again: `candidate`
+7. You should now be able to see the files of the codebase on the left side of the screen. Any code changes you make and save, your interviewers will be able to see.
 
-If Windows Machine doesn't have an OpenSSH compatible SSH client, follow this guide to install one via Windows Settings Note: This is not typical. Try running set up without this step:
-   - Install [an OpenSSH compatible SSH client](https://aka.ms/vscode-remote/ssh/supported-clients).
+If your Windows machine doesn't have an OpenSSH compatible SSH client, follow this guide to install one via Windows Settings: (Note: This is not typical. Try running setup without this step.)
+   - Install [an OpenSSH compatible SSH client](https://aka.ms/vscode-remote/ssh/supported-clients)
 
-# Disconnecting after interview:
-- Simply exit the VS Code terminal, and you will be disconnected from the ssh client.
+# Disconnecting After the Interview
+Simply exit the VS Code terminal, and you will be disconnected from the SSH client.
 
-# Helpful Debugging Commands:
+# Helpful Debugging Commands
 
-- To run Rails Console:
+- To run the Rails console:
 
 ```docker exec -it $( docker ps | grep homechef_mealhand | awk "{print \$1}" | head -n 1 ) rails c```
 
-- To attach a shell to the mealhand container in order to use ```binding.pry```:
+- To attach a shell to the container in order to use `binding.pry`:
```

Comment:
I would probably argue that this should say `mealhand` container just to be clear on which container we are attaching the shell too. This can also be done using the Docker extension that will already be on the VSCode of the EC2 instance, just in case for some weird reason this command isn't working by right clicking on the mealhand container and clicking "attach shell"

Reply:
I agree.